index.js

```diff
@@ -0,0 +1,60 @@
+const { Bot } = require('grammy');
+
+const bot = new Bot('5532232117:AAG_8EgKVkKd04srN7B5aHLKatC3biacIHc');
```

Comment:
I wouldn't recommend you to do this. Use env variables and remove all secrets from github that may compromise you

Reply:
In addition, you must revoke your token and generate a new one using BotFather with the .env file.

Reply:
Done